src/components/avatars/Avatar.stories.js

```diff
@@ -0,0 +1,90 @@
+// import MyButton from './Button.vue';
+import DdAvatar from "./index.vue"
+import { action } from "@storybook/addon-actions"
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Example/DdAvatar',
```

Comment:
Title should be Avatar just

Reply:
Title of avatar is changed 
<img width="964" alt="image" src="https://user-images.githubusercontent.com/85391382/217444399-5128d6b9-77d7-42c8-896b-ad60c1f89983.png">